core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CompareApplications.scala

```diff
@@ -32,7 +32,8 @@ class CompareApplications(apps: Seq[ApplicationInfo]) extends Logging {
   def findMatchingStages(): (Seq[CompareProfileResults], Seq[CompareProfileResults]) = {
     val normalizedByAppId = apps.map { app =>
       val normalized = app.sqlPlans.mapValues { plan =>
-        SparkPlanInfoWithStage(plan, app.accumIdToStageId).normalizeForStageComparison
+        SparkPlanInfoWithStage(plan,
+          app.stageManager.reduceAccumMapping()).normalizeForStageComparison
```

Comment:
`reduceAccumMapping()` is a hack to get the generateDot to work with the 1-to-M map.
GenerateDot is rarely used and fixing the implementation to be 1-to-M is going to bloat the PR